roles/openldap/tasks/setup_check_password.yml

```diff
@@ -0,0 +1,56 @@
+- name: install build requirements
+  apt: name={{ item }} state=installed
+  with_items: "{{ openldap.openldap_dev_pkgs }}"
+
+- name: create a temporary working directory
+  command: /bin/mktemp -d
+  register: working_dir
+
+- name: copy pre-built check_password library
+  copy: src=check_password/check_password.so.{{ ansible_distribution }}-{{ ansible_distribution_version }}-{{ ansible_userspace_bits }}
+        dest=/usr/lib/ldap/check_password.so
+  when: openldap.build_check_password_from_source == False
+  ignore_errors: True
+  register: prebuilt_check_passwd_available
+
+- name: get openldap source
+  git: repo={{ openldap.openldap_source_git }}
+       dest={{ working_dir.stdout }}/openldap
+       accept_hostkey=True
+  ignore_errors: True
+  when: openldap.build_check_password_from_source == True or prebuilt_check_passwd_available|failed
+
+- name: get check_password source
+  copy: src=check_password/trunk 
+        dest={{ working_dir.stdout }}/check_password
+  when: openldap.build_check_password_from_source == True or prebuilt_check_passwd_available|failed
+
+- name: build openldap headers
+  shell: ./configure && make chdir={{ working_dir.stdout }}/openldap
+  when: openldap.build_check_password_from_source == True or prebuilt_check_passwd_available|failed
+
+- name: build check_password extension
+  shell: make install CONFIG="/etc/openldap/check_password.conf" LDAP_INC="-I{{ working_dir.stdout }}/openldap/include/ -I{{ working_dir.stdout }}/openldap/servers/slapd" CRACKLIB='' CRACKLIB_OPT='' CRACKLIB_LIB='' LIBDIR='/usr/lib/ldap' chdir={{ working_dir.stdout }}/check_password/trunk
+  when: openldap.build_check_password_from_source == True or prebuilt_check_passwd_available|failed
```

Comment:
There is no prebuilt package, can we build a package and put it in any existing repo that ursula uses?